contributing/code/conventions.rst

```diff
@@ -111,3 +111,21 @@ ready to cope with them (by adding a custom error handler like the one used by
 the Web Debug Toolbar or by the PHPUnit bridge).
 
 .. _`@-silencing operator`: https://php.net/manual/en/language.operators.errorcontrol.php
+
+When deprecating a whole class the error should be placed between namespace and
+use declarations, like in this example from ArrayParserCache_::
```

Comment:
We always use additional backticks to enclose words that are part of a link role (even if it's just one word).